template/carousel/carousel.html

```diff
@@ -1,6 +1,6 @@
 <div ng-mouseenter="pause()" ng-mouseleave="play()" class="carousel" ng-swipe-right="prev()" ng-swipe-left="next()">
     <ol class="carousel-indicators" ng-show="slides.length > 1">
-        <li ng-repeat="slide in slides track by $index" ng-class="{active: isActive(slide)}" ng-click="select(slide)"></li>
+        <li ng-repeat="slide in slides | orderBy:'index' track by $index" ng-class="{active: isActive(slide)}" ng-click="select(slide)"></li>
```

Comment:
So my concern is if the user is doing something like
```
<slide ng-repeat="slide in slides | orderBy: customOrderFn">
...
</slide>
```
I'm not sure whether that custom order would be respected by this implementation.